lib/salus/scanners/base.rb

```diff
@@ -178,10 +179,201 @@ def validate_file_option(keyword)
 
       return true if config_dir.include?(Dir.pwd) # assumes the current directory is the proj dir
 
-      report_warn(:scanner_misconfiguration, "Expecting #{@config.fetch(keyword)} defined by \
+      report_warn(:scanner_misconfiguration, "Expecting #{value} defined by \
                                               #{keyword} to be a dir in the project repo but was \
                                                located at '#{config_dir}' instead.")
       false
     end
+
+    def create_flag_option(keyword:, value:, prefix:, suffix:)
+      return '' unless validate_bool_option(keyword, value)
+
+      if value.to_s.downcase == "true"
+        "#{prefix}#{keyword}#{suffix}"
+      else
+        ''
+      end
+    end
+
+    def create_bool_option(keyword:, value:, prefix:, between:, suffix:)
```

Comment:
nit: Perhaps a better name for `between` like `keyval_separator`

Reply:
How about just separator?

Reply:
nit: `suffix` like `arg_separator`

nit: `prefix` like `option_separator`

Probably fine though

Reply:
Well, there's a lot of separators here though. Naming is hard enough and this only has to be remembered once anyways that it isn't a huge deal